packages/testing/src/test/Helpers.test.ts

```diff
@@ -2,102 +2,112 @@
  *--------------------------------------------------------------------------------------------*/
-
 import { expect } from "chai";
-import { HierarchyCacheMode, initialize, PresentationTestingInitProps, terminate } from "../presentation-testing/Helpers";
-import * as sinon from "sinon";
-import { Presentation as PresentationBackend } from "@itwin/presentation-backend";
-import { Presentation as PresentationFrontend } from "@itwin/presentation-frontend";
 import * as rimraf from "rimraf";
+import * as sinon from "sinon";
 import { Guid } from "@itwin/core-bentley";
 import { NoRenderApp } from "@itwin/core-frontend";
+import { Presentation as PresentationBackend, PresentationProps } from "@itwin/presentation-backend";
+import { Presentation as PresentationFrontend } from "@itwin/presentation-frontend";
+import { HierarchyCacheMode, initialize, PresentationTestingInitProps, terminate } from "../presentation-testing/Helpers";
 
 describe("Helpers", () => {
   describe("initialize", () => {
+    let backendInitializationStub: sinon.SinonStub;
+    let frontendInitializationStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      backendInitializationStub = sinon.stub(PresentationBackend, "initialize");
+      frontendInitializationStub = sinon.stub(PresentationFrontend, "initialize");
+      sinon.stub(PresentationBackend, "terminate");
+      sinon.stub(PresentationFrontend, "terminate");
+    });
+
     afterEach(async () => {
       await terminate();
     });
 
     it("initializes PresentationBackend and PresentationFrontend on initialize", async () => {
-      const backendInitializationSpy = sinon.spy(PresentationBackend, "initialize");
-      const frontendInitializationSpy = sinon.spy(PresentationFrontend, "initialize");
-
       await initialize();
-      expect(backendInitializationSpy.calledOnce);
-      expect(frontendInitializationSpy.calledOnce);
+      expect(backendInitializationStub).to.be.calledOnce;
+      expect(frontendInitializationStub).to.be.calledOnce;
     });
 
     it("does not initialize PresentationBackend and PresentationFrontend on second call to initialize", async () => {
-      const backendInitializationSpy = sinon.spy(PresentationBackend, "initialize");
-      const frontendInitializationSpy = sinon.spy(PresentationFrontend, "initialize");
-
       await initialize();
-      expect(backendInitializationSpy.calledOnce);
-      expect(frontendInitializationSpy.calledOnce);
+      expect(backendInitializationStub).to.be.calledOnce;
+      expect(frontendInitializationStub).to.be.calledOnce;
 
       sinon.resetHistory();
 
       await initialize();
-      expect(backendInitializationSpy.notCalled);
-      expect(frontendInitializationSpy.notCalled);
+      expect(backendInitializationStub).to.not.be.called;
+      expect(frontendInitializationStub).to.not.be.called;
     });
 
     it("initializes PresentationBackend and PresentationFrontend with provided props", async () => {
```

Comment:
the test name says it checks presentation frontend initialized too, although the test itself doesn't do the check

Reply:
Added the check to the test, now the name should be correct.